examples/bit_packing/ex_quantize_and_pack.py

```diff
@@ -49,39 +50,42 @@
 apply_quantization_config(model, config)
 
 # create dataset
+dataset = load_dataset(dataset_name, split="train[:128]")
 tokenizer = AutoTokenizer.from_pretrained(model_name)
-data_args = DataTrainingArguments(
-    dataset=dataset_name,
-    max_seq_length=max_seq_length,
-    pad_to_max_length=pad_to_max_length,
-)
-dataset_manager = TextGenerationDataset.load_from_registry(
-    data_args.dataset,
-    data_args=data_args,
-    split=split,
-    tokenizer=tokenizer,
-)
-calib_dataset = dataset_manager.tokenize_and_process(
-    dataset_manager.get_raw_dataset()
-)
+
+
+def tokenize_function(examples):
+    return tokenizer(
+        examples["output"], padding=False, truncation=True, max_length=1024
+    )
+
+
+tokenized_dataset = dataset.map(tokenize_function, batched=True)
+
 data_loader = DataLoader(
-    calib_dataset, batch_size=1, collate_fn=DefaultDataCollator(), sampler=RandomSampler(calib_dataset)
+    tokenized_dataset,
+    batch_size=1,
+    collate_fn=DefaultDataCollator(),
+    sampler=RandomSampler(tokenized_dataset),
```

Comment:
This is the default, so it should be safe to remove for simplicity of the example